.github/workflows/node-test.yml

```diff
@@ -1,41 +1,26 @@
-name: Node
+name: Tests
 
 on:
-  pull_request:
   push:
     branches:
-      - master
-      - stable*
+      - '*'
 
 jobs:
-  build:
+  tests:
```

Comment:
Please keep the Node/test Ci output like other repositories :)
Also, keep pull_requests

```yml
name: Node

on:
  pull_request:
  push:
    - master

jobs:
  test:
```